Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -755,4 +760,157 @@ lottieQRView config push =
+          , infoCardView config (if config.isDriver then "VERTICAL" else "HORIZONTAL") $ getRentalRideInfoCardOdometerView config "RideCompletedCardImage4" config.rentalRideConfig.endRideOdometerImage (config.rentalRideTextConfig.rideEnd) (config.rentalRideTextConfig.rideEndedAt) (config.rentalRideConfig.rideEndODOReading)
+        ]
+    ]
+
+getRentalRideInfoCardOdometerView :: forall w. Config -> String -> String -> String -> String -> String -> (InfoCardConfig)
+getRentalRideInfoCardOdometerView config image renderImage heading heading' subHeading1  = 
+  { id : image, 
+    margin : (MarginBottom 0 ), 
+    height : WRAP_CONTENT, 
+    width : V ((screenWidth unit) / 2) ,
+    image : {
+      visibility : GONE, 
+      height  : V 72, width : V 110, 
+      renderImage : if config.isDriver then (renderImage) else ""
+      } , 
+    heading : {
+      text : if config.isDriver then heading else heading' , 
+      color : Color.black700 , 
+      fontStyle : FontStyle.body1 TypoGraphy, 
+      visibility : VISIBLE
+      }, 
+    subHeading1 : {
+      text :  subHeading1, 
+      color : Color.black800 , 
+      fontStyle : FontStyle.body2 TypoGraphy, 
+      visibility : VISIBLE
+      }, 
+    subHeading2 : {
+      text : "" , 
+      color : Color.black600 , 
+      fontStyle : FontStyle.body2 TypoGraphy, 
+      visibility : GONE
+      }}
+
+getRentalRideInfoCardView :: forall w. Config -> String -> String -> String -> String -> (InfoCardConfig)
+getRentalRideInfoCardView config image heading subHeading1 subHeading2 =
+  { id : image,
+    margin : (MarginBottom 0 ), 
+    height : WRAP_CONTENT, 
+    width : V ((screenWidth unit) / 2) ,
+    image : {
+      visibility : GONE , 
+      height  : V 100, 
+      width : V 100, 
+      renderImage : ""
+      }, 
+    heading : {
+      text : heading, 
+      color : Color.black700, 
+      fontStyle : FontStyle.body1 TypoGraphy, 
+      visibility : VISIBLE
+      }, 
+    subHeading1 : {
+      text : subHeading1, 
+      color : Color.black800, 
+      fontStyle : FontStyle.body2 TypoGraphy, 
+      visibility : VISIBLE
+      }, 
+    subHeading2 : {
+      text : " / " <> subHeading2, 
+      color : Color.black600, 
+      fontStyle : FontStyle.body2 TypoGraphy, 
+      visibility : VISIBLE
+      }}
+
+
+infoCardView :: forall w. Config -> String -> InfoCardConfig -> PrestoDOM (Effect Unit) w
+infoCardView config orientation' infoCardConfig = 
+  linearLayout
+  [ height infoCardConfig.height
+  , weight 1.0
+  , width MATCH_PARENT
+  , margin infoCardConfig.margin
+  , gravity LEFT
+  , orientation if orientation' == "VERTICAL" then VERTICAL else HORIZONTAL
```

Comment:
use boolToVisibility from Mobility.Prelude module

Reply:
we are not setting visibility